### ISSUE TITLE
Shortcut Symbol is sometimes removed

### ISSUE THREAD

**Issue by StealthEngine** [cataclysmbnteam/Cataclysm-BN#4058]
### Describe the bug

I usually adjust the shortcut symbol in my gun for reloading convenience.
Almost it works well, but sometimes shortcut symbol is removed automatically.

### Steps To Reproduce

I don't know why it is appeared, so I can't suggest the steps.
Anyway, In my cases,
1. Adjust shortcut symbol "r" in Marlin 39A.
2. Shot the enomy.
3. Reloading ammos just keep pressing "r"
4. Somtimes symbol "r" is removed

### Screenshots

Below is picture adjusting symbol "r" in my gun.
![063](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/68223808/ca6dea3d-8103-40e8-b9a7-fa3afd1b2fc7)

Sometimes, symbol is removed. You can see the no symbol in my gun even if "r" is activated in yellow.
![065](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/68223808/b021fc76-8b92-4648-b499-4a3d19e64553)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22621.2861 (22H2)
- Game Version: 0.5 [64-bit]
- Graphics Version: Tiles
- LAPI Version: 1
- Game Language:  []
- Mods loaded: [
    밝은 밤 [bn],
    NPC 욕구 제거 [no_npc_food],
    철도역 금지 [No_Rail_Stations],
    진균 성장 제한 [limit_fungal_growth],
    Fuji의 군사 직업 팩 [fuji_mpp],
    기술 성장 비례 능력 성장 [StatsThroughSkills],
    살해 수 반영 능력 성장  [stats_through_kills],
    C.R.I.T. 확장팩 [crt_expansion],
    희망 없는 세계 [no_hope],
    매직클리즘 [magiclysm],
    공간이동 [teleportation_tech],
    총기 제조 팩 [craftgp],
    신화 속의 무술 [MMA]
]

**Comment by StealthEngine** [comment]
I got the point. Using crossbow is better. Steps to reproduction is :
1. adjust any shortcut symbol in crossbow.
2. shoot it and get the ammo.
3. shortcut symbol is removed.

**Comment by Ignaramico** [comment]
isnt that because the name gets changed?, i remember making an issue where a favorited item messed with the autopickup

**Comment by StealthEngine** [comment]
No. I too doubted that at first, because name is changed after reloading.
But I found it after pick up something.

Below is before pick up something. I adjust the shortcut symbol "r" in my compound crossbow and wielding it.
<img width="457" alt="001" src="https://github.com/cataclysmbnteam/Cataclysm-BN/assets/68223808/e8b8ebbb-6a4a-4977-9bf8-ad375c45dc58">

After pick up anything, symbol is removed.
<img width="499" alt="002" src="https://github.com/cataclysmbnteam/Cataclysm-BN/assets/68223808/3da53815-1494-4ae7-b96a-bb7814109562">

**Comment by StealthEngine** [comment]
I play ver.6a10d32, 2024-01-14 updated, this is still happened.
Also, I found that this issue is appeared while item interaction.
Even I don't get the new item, shortcut symbol is removed.
Just press "e" for trying get item, shortcut symbol is removed.

I find the temporary solution for this; adjusting shortcut symbol in my inventory, not wielding it.
